packages/block-library/src/post-featured-image/edit.js

```diff
@@ -55,14 +107,38 @@ function PostFeaturedImageDisplay( {
 					/>
 				</PanelBody>
 			</InspectorControls>
+			<BlockControls>
+				{ !! media && (
+					<>
+						<ToolbarGroup>
```

Comment:
It's a single item, so it might not need a group. Although, I see that `MediaReplaceFlow` is also wrapped with the group so 🤷‍♂️

Reply:
🤷‍♂️ Any input from others maybe? In general I don't think it's something to worry us much in either case.

Reply:
This also applies the styles for `borders` which I wanted to separate semantically for the two controls. In other way I should add css probably, unless there is an option or class that I'm not aware of.... @jasmussen

Reply:
Can you clarify the question? Is this about the styling of the control in the sidebar?

Reply:
Is there a `css class` for applying the right border in a `ToolbarButton` to look like if it was inside `ToolbarGroup`. Do you think it would be preferable to add custom css for this border instead wrapping it in `ToolbarGroup` even if it's a single ToolbarButton?

Reply:
![Screenshot(1)](https://user-images.githubusercontent.com/16275880/100094305-7be14b80-2e61-11eb-969a-e156f37b9c41.png)

Reply:
Oh, thanks for clarifying! I actually wonder if we should replace that icon with a label instead, "Unset" or "Clear". It feels weird to have an icon next to the "Replace" label, and the icon isn't super clear.

I will defer to you and other developers on what the best approach is here — I would agree that "Clear" and "Replace" should either be in the same toolbar-group, or appear to be.

Reply:
>the icon isn't super clear.
hm.. this is the icon for Post Featured Image with the crossing line.. It seems clear to me but of course this is subjective..

>feels weird to have an icon next to the "Replace" label,

That's why I think a border is needed (if we keep it). Also `Replace` exists next to icons in `Image` blocks.

>I would agree that "Clear" and "Replace" should either be in the same toolbar-group, or appear to be.
So you suggest there shouldn't be a border there and change the icon to a `Clear` label, correct? I. can change it to whatever makes most sense.

Reply:
You can keep the group if it impacts visual aspects, but it's just strange to see it because in real life a group of items is definitely more than one 😄 

Aside: should it be maybe "Replace" and then "Unset" in the toolbar?  It also prompts the question of whether it should be the text for both (or icons) for consistency. A separate concern is whether those controls shouldn't be inlined in the block content. I think we have a very similar issue with `RichText` and we discussed several times that it would be better to have it more contextual.

Reply:
> So you suggest there shouldn't be a border there and change the icon to a Clear label, correct? I. can change it to whatever makes most sense.

This feels good to me:

![Untitled-1](https://user-images.githubusercontent.com/1204802/100096259-1048af80-2e5c-11eb-88cd-e77f52be15d8.png)

In general I would recommend trying to reduce as many borders as we can. See also https://github.com/WordPress/gutenberg/issues/26411